lambda-code/reliability/lib/dataLayer.ts

```diff
@@ -198,10 +198,15 @@ export function extractFileInputResponses(submission: FormSubmission) {
             if (current.type === "fileInput") {
               const response = submission.responses[element.id];
 
-              const fileInputPath = Array.isArray(response) && response[currentIndex];
-              if (fileInputPath !== "" && typeof fileInputPath === "string") {
-                return [...acc, fileInputPath];
```

Comment:
Been wondering why we had this code in place before (the check for empty string and type check). This is for when  the repeating set has optional file input components. I tested it locally and the updated code fails to handle such form response.

```
{ '2': '', '3': '', '4': [ { '0': 'test', '1': '' }, { '0': 'test2', '1': 'form_attachments/2024-08-13/0f5db5a4-efa6-4abf-9f7e-3fdf19c7eacf/Screenshot 2024-07-30 at 8.23.05 AM.png' } ] }
```

Reply:
aha, ok will update